alacritty/src/event.rs

```diff
@@ -297,16 +297,20 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
 
         #[cfg(unix)]
         let args = {
+            // Use working directory of controlling process, or fallback to initial shell.
+            let mut pid = unsafe { libc::tcgetpgrp(tty::master_fd()) };
+            if pid == -1 {
```

Comment:
```suggestion
            if pid < 0 {
```

Reply:
Should never happen, but better safe than sorry.

Reply:
it's just a matter of proper error handling.